skills/declarative/calendarSkill/language-generation/en-us/event.en-us.lg

```diff
@@ -259,7 +259,9 @@
     -
 
 # EventAttendeeContainer(event)
-- IF: ${ event.attendees != null && count(event.attendees) > 1 }
+- IF: ${ event.Response == 'organizer' }  
+    -
+- ELSEIF: ${ event.attendees != null && count(event.attendees) > 1 }
```

Comment:
Dumb question - is event.attendees > 1 because the organizer is always person 0?